cluster_kwargs.yaml

```diff
@@ -32,19 +32,19 @@ parquet_cluster:
   worker_vm_types: [m5.xlarge]  # 4 CPU, 16 GiB
 
 # For tests/workflows/test_embarrassingly_parallel.py
-embarrassingly_parallel_cluster:
+embarrassingly_parallel:
   n_workers: 100
   worker_vm_types: [m6i.xlarge] # 4 CPU, 16 GiB (preferred default instance)
   backend_options:
     region: "us-east-1"  # Same region as dataset
 
-# For tests/workflows/test_optuna.py
-optuna_cluster:
+# For tests/workflows/test_xgboost_optuna.py
```

Comment:
Just making sure this is just a change in the comment, right? Because if  the name of the test changed, then there should be a DB migration.

Reply:
Yeah, this is just a comment update. The test module is actually already named `test_xgboost_optuna.py`, I just forgot to update this comment in the PR that added the test.